components/network/setup.ftl

```diff
@@ -192,25 +225,48 @@
             /]
           [/#if]
 
+          [#-- format port ranges as string type --]
+          [#if sourcePort?is_string]
+            [#local sourcePort = sourcePort?replace("any", "*")]
+          [#else]
+            [#local sourcePort = sourcePort.PortRange.From?string?replace(",", "") + "-" + sourcePort.PortRange.To?string?replace(",", "")]
```

Comment:
Where did the commas come in? Is that freemarker making numbers look pretty? You can turn that of with `?c` at the end of the var. It converts it into computer language

Reply:
Yep it was due to freemarker converting the numbers to strings and adding formatting. Mike and I discussed it yesterday and I ended up not needing ?c for that instance, but I neglected to incorporate it here. Thanks - will add during the Gateway component.